draft-ietf-quic-qlog-main-schema.md

```diff
@@ -1514,86 +1514,70 @@ still provide adequate output for incomplete logs.
 
 # Security and privacy considerations {#privacy}
 
-Protocols such as TLS {{?RFC8446}} and QUIC {{?RFC9000}} provide varying degrees
-of secure protection for the wire image {{?RFC8546}}. There is inevitably
-tension between security and observability, when logging can reveal aspects of
-the wire image that would ordinarily be protected. This tension equally applies
-to any privacy considerations that build on security properties, especially if
-data can be correlated across data sources.
-
-Any data that is determined to be necessary for a use case at hand could be
-logged or captured. As per {{?RFC6973}}, operators must be aware that such data
-will be at risk of compromise and this, in turn, risks the privacy of all
-entities that participate in logged internet protocol exchanges. Entities that
-would otherwise expect protocol features to maintain the privacy of their data,
-can unknowingly be subject to broader privacy risks. This circumvents their
-agency for responding to such risks.
+Protocols such as TLS {{?RFC8446}} and QUIC {{?RFC9000}} offer secure protection
+for the wire image {{?RFC8546}}. Logging can reveal aspects of the wire image
+that would ordinarily be protected, creating tension between observability,
+security and privacy, especially if data can be correlated across data sources.
+
+Depending on the observability use case any data could be logged or captured. As
+per {{?RFC6973}}, operators must be aware that such data could be compromised,
+risking the privacy of all participants. Entities that expecti internet protocol
+features to ensure data privacy might unknowingly be subject to broader privacy
+risks, undermining their ability to assess or respond effectively.
 
 ## Data at risk
 
-qlog operators and implementers should be mindful of the security and privacy
-risks inherent in handling qlog data. This includes but is not limited to
-logging, storing, or using the data. Data might be considered as non-sensitive,
-potentially-sensitive, or sensitive; applying the considerations in this section
-may produce different risks depending on the nature of the data itself, or its
-handling. However, in many cases the largest risk factors arise from data that
-can be considered as potenially-sensitive or sensitive.
+qlog operators and implementers need to consider security and privacy risks when
+handling qlog data, including logging, storage, usage and more. Data can be
+categorized as non-sensitive, potentially-sensitive, or sensitive. The
```

Comment:
```suggestion
handling qlog data, including logging, storage, usage, and more.  The
```
Optionally.  The classification thing doesn't add to the narrative.

Reply:
@rmarx any strong feelings here since it is your text?

Reply:
It's fine for me to remove this, but then there's another sentence just a little bit further that also mentions "potentially-sensitive and sensitive" that probably should be cut as well.